presto-docs/README.md

```diff
@@ -0,0 +1,73 @@
+# presto-docs - Presto Documentation
```

Comment:
Add a link in main README to make this document more discoverable.

Reply:
I agree with the notion to add a link. However I am not sure where in the root level README would be a good place. Esp if we do this approach of a readme in each module..

Reply:
Done now,.